Dockerfile

```diff
@@ -5,8 +5,8 @@ LABEL maintainer="Rick Yu <cosmtrek@gmail.com>"
 ENV GOPATH /go
 ENV GO111MODULE on
 
-COPY . /go/src/github.com/cosmtrek/air
-WORKDIR /go/src/github.com/cosmtrek/air
+COPY . /go/src/github.com/zhb127/air

```

Comment:
@zhb127 Why are these links also being updated ..? I would be happy if this PR was merged, but it looks like these need to be fixed.

Reply:
Sorry, I've restored the code.

Reply:
I originally thought this feature wasn't accepting , so I made modifications for the forked repository.
sorry @flada-auxv

Reply:
No, you don't need to apologize. Thank you for your response!